[add]SaveAndLoadスクリプトの追加、Resourcesフォルダの追加

629Kitamura opened the pull request:
Resourcesフォルダの中のSeveDataの各数値と書き換えるとゲームスタート時にその数値が適用されます。 Facilityスクリプトの_nameをpublicまたは外から参照できるようにしてほしいです。_nameの書き換えは発生しません。

629Kitamura committed:
[add]SaveAndLoadスクリプトの追加、Resourcesフォルダの追加
Resourcesフォルダの中のSeveDataの各数値と書き換えるとゲームスタート時にその数値が適用されます。
Facilityスクリプトの_nameをpublicまたは外から参照できるようにしてほしいです。_nameの書き換えは発生しません。

629Kitamura commented:
煮詰まったので一旦プルリクエストします。
今はFacilityの_nameと_ownedNumを保存していますが他にFacilityで保存したいデータはありますか？
UpGradeはなにをどのように保存すればいいかわかりません。指示をもらえると助かります。

KentoMatsuzaki commented:
確認遅れて申し訳ないです！（17日は一日外にいて気付かなかった......）
保存したい内容としては、仕様書の転生の項目にある「クッキーの数」「施設」「アップグレード」かな。
あとは60秒に一回のオートセーブとロード機能お願いします。

KentoMatsuzaki commented:
アップグレードは全てにbool型の所有を意味する変数「_isOwned」を持たせようと思っています。

629Kitamura committed:
[add,fix,del]SaveManagerとAutoSaveとそれぞれをアタッチしたプレハブの追加、SaveAndLoadの削除、SeveDataの一部書き換え

629Kitamura committed:
[add]LoadManagerとそれをアタッチしたプレハブの追加

629Kitamura committed:
[fix]UTF-8に修正

629Kitamura committed:
[fix]UTF-8に修正

629Kitamura committed:
Merge branch 'LoadManager' into SaveAndLoad

629Kitamura committed:
[fix]SaveDataクラスにcurrentPrice(セーブ時の購入金額)を追加、AutoSaveの修正
Facilityの_currentPriceをロード時にセーブデータで書き換えるようにしました。
FacilityのStartで_currentPriceを初期化していますがそこを消すと上書きされずに表示できます。

629Kitamura committed:
[fix]isUpGradeの代わりにcurrentUpgradeFactorをセーブ＆ロードするように変更。

629Kitamura committed:
[fix]コメントの追加
